packages/perseus/src/widgets/__stories__/on-screen-keypad.stories.jsx

```diff
@@ -0,0 +1,118 @@
+// @flow
+import {Keypad} from "@khanacademy/math-input";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import {ItemRenderer} from "../../index.js";
+import KeypadContext from "../../keypad-context.js";
+import {expressionItem3} from "../__testdata__/expression_testdata.js";
+
+type StoryArgs = {||};
+
+type Story = {|
+    title: string,
+|};
+
+export default ({
+    title: "Perseus/Demos/On Screen Keypad",
+}: Story);
+
+const Content = (): React.Node => {
+    return (
+        <KeypadContext.Consumer>
+            {({keypadElement, setRenderer, scrollableElement}) => (
+                <>
+                    <ItemRenderer
+                        ref={setRenderer}
+                        problemNum={0}
+                        apiOptions={{
+                            customKeypad: true,
+                            onFocusChange: (
+                                newFocusPath,
+                                oldFocusPath,
+                                keypadElement,
+                                focusedElement,
+                            ) => {},
+                        }}
+                        item={expressionItem3}
+                        savedState={null}
+                    />
+                    <div id="workarea" />
+                    <div id="hintsarea" />
+                </>
+            )}
+        </KeypadContext.Consumer>
+    );
+};
+
+const Footer = (): React.Node => {
+    return (
+        <View
+            style={[
+                styles.keypadContainer,
+                {
+                    // NOTE: in webapp we normally get this from problemProgress
+                    // from the exercise state store.
+                    height: 240,
```

Comment:
Since this is static, is there any reason it can't be in the `keypadContainer` style?

Reply:
I'm not a 100% sure this is static.  I have to look at the code in webapp to see if the value is always the same or not.